site/en/docs/privacy-sandbox/attribution-reporting-event-guide/index.md

```diff
@@ -0,0 +1,946 @@
+{% endColumn %}
+
+{% Column %}
+{% Img src="image/O2RNUyVSLubjvENAT3e7JSdqSOx1/u3Gfm4ebwIr9NREfq5eE.png", alt="ALT_TEXT_HERE", width="400", height="300" %}
+{% endColumn %}
+
+{% endColumns %}
+
+## Enable the API in your browser
+
+If you are developing locally or testing out a deployed demo app in your browser, it's enough to enable the API **in your browser**.
+
+{% Details %}
+{% DetailsSummary %}
+
+How to enable the API in your browser
+
+{% endDetailsSummary %}
+
+Enable the API via a _flag_. A flag is a special setting in your browser.
+
+1. Type or paste `chrome://flags/#enable-experimental-web-platform-features` in Chrome's URL bar. This works on mobile, too.
+2. Click **Enabled.**
+3. Restart Chrome.
+4. Ensure that Privacy Sandbox features are turned **on** at `chrome://settings/privacySandbox`. If
+   they're turned off, this would override your flag and disable the Attribution Reporting API.
+
+<figure>
+   {% Img src="image/O2RNUyVSLubjvENAT3e7JSdqSOx1/bKfzMxqL4gpbDDw3NLDc.png", alt="ALT_TEXT_HERE", width="400", height="261" %}
+   <figcaption>Turn on the flag to enable the API in your browser.</figcaption>
+</figure>
+
+{% endDetails %}
+
+## Enable the API for end users (origin trial)
+
+To enable the API for all your site's users⏤except those who have chosen to opt-out of Privacy Sandbox features⏤register your site for an _origin trial_. You can also register for an origin trial if your code is embedded in sites you don't own, as third-party code.
+
+{% Aside 'key-term' %}
+An _origin trial_ (sometimes called an _experiment_) is a way to test a new or experimental web platform feature, and give feedback to the web standards community on the feature.
+Learn more in [Getting started with Chrome's origin trials](/blog/origin-trials/).
+{% endAside %}
+
+### Design your origin trial
+
+{% Details %}
+{% DetailsSummary %}
+
+How to design your origin trial
+
+{% endDetailsSummary %}
+
+Before you set up an origin trial, set time aside to design your experiment. Ask yourself and your team: _What is our goal with this experiment?_
+
+If your goal is to understand the API mechanics, run your experiment as follows:
+
+- Track conversions.
+- See how you can assign different values to attribution trigger events (conversions).
+- Look at the reports you're receiving.
+
+If your goal is to see how well the API satisfies basic use cases, run your experiment as follows:
+
+- Track conversions.
+- Look at the count of reports you're receiving.
+- [Recover the corrected conversion
+  count](#recover-the-corrected-conversion-count). Optionally, if you want to try something more advanced, tweak the noise correction script.
+  For example, try different groupings to see what sizes are necessary for the noise to be negligible.
+- Compare the corrected count of conversions with source-of-truth data, that is cookie-based
+  measurement data.
+
+{% endDetails %}
+
+### Set up an origin trial
```

Comment:
Same as above, subheading seems redundant, especially if "How to set up an origin trial" can be a part of the TOC.

Reply:
(Same as above)